pkg/base/linuxutils/lsblk/lsblk.go

```diff
@@ -178,7 +173,7 @@ func (l *LSBLK) SearchDrivePath(drive *api.Drive) (string, error) {
 	vid := drive.VID
 	pid := drive.PID
 	for _, l := range lsblkOut {
-		if strings.EqualFold(l.Serial, sn) && strings.EqualFold(l.Vendor, vid) &&
+		if strings.EqualFold(l.Serial, sn) && strings.Contains(l.Vendor, vid) &&
```

Comment:
how about the pid handling ?  lsblk returned "HGST HUS728T8TALE6L0",  but the drive CR value is from drivemgr,  which is "HGST HUS728T8TAL". 
and also is there possibility that Serinal need same handling with Vendor,  as Serialnumber may also contain some trailing space.

Reply:
We are observing failure only for the Vendo ID. Since the serial number reported by both HAL and lsblk are the same, we should be good with the existing implementation. We need a quick resolution due to the time constraint, however I agree that we need to have a relook at this area of code in the next release cycle

Reply:
do you mean for the pid I mentioned above,  for different length strings,   strings.EqualFold will return true in the case ?

Reply:
I think we should only ignore the trailing space in this JIRA. It is not safe to use "string.contains(A, B)" as it will also return TRUE for many other cases. For example:  A = "testing   "    B="ing"